src/gui/transferlistsortmodel.cpp

```diff
@@ -61,27 +71,12 @@ namespace
     }
 
     template <typename T>
-    concept Validateable = requires (T t) { {t.isValid()} -> std::same_as<bool>; };
```

Comment:
You should also remove the concepts header.

Reply:
> You should also remove the concepts header.

Done.